sopel/db.py

```diff
@@ -595,28 +648,36 @@ def merge_nick_groups(self, first_nick: str, second_nick: str):
         """
         first_id = self.get_nick_id(first_nick, create=True)
         second_id = self.get_nick_id(second_nick, create=True)
-        session = self.ssession()
-        try:
+        with self.session() as session:
             # Get second_id's values
-            res = session.query(NickValues).filter(NickValues.nick_id == second_id).all()
+            results = session.execute(
+                select(NickValues).where(NickValues.nick_id == second_id)
+            ).scalars()
```

Comment:
Just looking at https://docs.sqlalchemy.org/en/14/changelog/migration_20.html#migration-orm-usage, it looks like they recommend .all() becomes .scalars().all()

Reply:
The method `all()` will return a `list`, which I don't need: I'll directly iterate over `results` the line below.